WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -27800,6 +27810,7 @@
 				CURRENT_PROJECT_VERSION = "${VERSION_LONG}";
 				DEFINES_MODULE = YES;
 				DEVELOPMENT_TEAM = PZYM8XX95Q;
+				"DEVELOPMENT_TEAM[sdk=iphoneos*]" = PZYM8XX95Q;
```

Comment:
I don't think this will cause any issues but is this an intended change?

Reply:
Ah yea, I should probably revert this.